### DISCUSSION TITLE
llama-index-core 0.10.45 issues

### DISCUSSION THREAD

**Discussion by wadave** [run-llama/llama_index#14174] [General]
'pip install llama-index-core==0.10.45' did not install the latest files from https://github.com/run-llama/llama_index/tree/main/llama-index-core/llama_index/core/utilities

These two toml files do not match:
https://github.com/run-llama/llama_index/blob/main/llama-index-core/pyproject.toml it shows 0.10.45 version 

https://github.com/run-llama/llama_index/blob/main/pyproject.toml it shows llama-index-core = "0.10.44"

Please check.

**Comment by cbornet** [comment]
Should be fixed by https://github.com/run-llama/llama_index/pull/14188

**Comment by wadave** [comment]
Thanks. how soon will it be published to pypi.
It still shows the old version (6/14/2024)
pip install llama_index-core==0.10.45
gives this error:
ERROR: pip's dependency resolver does not currently take into account all
the packages that are installed. This behaviour is the source of the
following dependency conflicts. llama-index 0.10.45 requires
llama-index-core==0.10.44, but you have llama-index-core 0.10.45 which is
incompatible.

**Reply by cbornet** [reply]
Well, a release will need to bump the version to 0.10.46.
So 0.10.45 is forever broken…

**Reply by cbornet** [reply]
0.10.46 is out.
You can upgrade.

**Comment by wadave** [comment]
It's fixed here:
https://github.com/run-llama/llama_index/pull/14198
